python-avd/pyavd/_eos_designs/shared_utils/node_type_keys.py

```diff
@@ -174,6 +173,39 @@
                 "l3": True,
             },
         },
+        {
+            "key": "ptp_leaf",
+            "type": "ptp_leaf",
+            "connected_endpoints": True,
+            "network_services": {
+                "l2": True,
```

Comment:
Do we really need L2 on all device types? Do you have a design I can look at?

Reply:
Yes PTP distribution leafs can also have l2 connected endpoints . Slide 13 here is an example where PTP leaf is also a media_leaf which has connected endpoints which can be a mix of both L2 and/or L3. https://docs.google.com/presentation/d/1XyV6eMJpGsNNP9bxZaxJ25e2VBK96W2rOyHbwZgKanw/edit#slide=id.g1a2c972f82e_0_204